src/additional-svelte-jsx.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace svelte.JSX {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	interface DOMAttributes<T> {
+		onoutclick?: MouseEventHandler<T> | undefined | null;
```

Comment:
this is how they recommend to add TS support for custom dom attributes: https://github.com/sveltejs/language-tools/blob/master/docs/preprocessors/typescript.md#im-using-an-attributeevent-on-a-dom-element-and-it-throws-a-type-error